include/amici/exception.h

```diff
@@ -18,8 +18,9 @@ class AmiException : public std::exception {
   public:
     /**
      * @brief Default ctor.
+     * @param first_frame Index of first frame to include
      */
-    AmiException();
+    AmiException(int const first_frame = 3);
```

Comment:
why not use 4 as this is what's used below (or not have any default)?

Reply:
If you do `throw AmiException()`, you need to drop 3 frames to get to the throwing frame.

If you do `throw AmiException("foo")`, you need to drop 4 frames to get to the throwing frame, because this delegates to `AmiException()`.